src/frontend/src/flows/promptUserNumber.ts

```diff
@@ -2,20 +2,21 @@ import { html, render, TemplateResult } from "lit-html";
 import { Ref, ref, createRef } from "lit-html/directives/ref.js";
 import { mkAnchorInput } from "../components/anchorInput";
 
-const pageContent = (
-  title: string,
-  userNumber: bigint | null,
-  callbacks: { onContinue: (ret: bigint) => void; onCancel: () => void }
-): { template: TemplateResult } => {
+const promptUserNumberTemplate = ({
+  title,
+  userNumber,
+  onContinue: onSubmit,
+  onCancel,
+}: {
+  title: string;
+  userNumber?: bigint;
```

Comment:
the `userNumber` used to be `biging` or `null` but becomes `bigint` or `undefined`. that's fine? no side effect?

Reply:
Side effects taken care of, see useRecovery and showcase!